vendor/bat-native-ads/src/bat/ads/internal/account/statement/next_payment_date_util.cc

```diff
@@ -70,10 +70,10 @@ base::Time CalculateNextPaymentDate(const base::Time& next_token_redemption_at,
   next_payment_date_exploded.month = month;
   next_payment_date_exploded.day_of_month =
       features::GetAdRewardsNextPaymentDay();
-  next_payment_date_exploded.hour = 0;
-  next_payment_date_exploded.minute = 0;
-  next_payment_date_exploded.second = 0;
-  next_payment_date_exploded.millisecond = 0;
+  next_payment_date_exploded.hour = 23;
```

Comment:
could this time be converted to next day for UTC+5 timezone?

Reply:
That is correct and expected